crates/drainer/src/utils.rs

```diff
@@ -100,6 +100,7 @@ pub async fn make_stream_available(
         .delete_key(stream_name_flag)
         .await
         .map_err(DrainerError::from)
+        .map(|_r| ())
```

Comment:
Please revert this change.

Reply:
@jeevaprakashdr This is still pending.

Reply:
I thought I have dropped a message,  on this.  WE would need the error part  else we get the following error 

![image](https://github.com/juspay/hyperswitch/assets/6055626/2fdcde35-960f-4985-b583-ffbb93fef55c)